src/main.ts

```diff
@@ -3,6 +3,6 @@ import { AppModule } from './app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
-  await app.listen(3000);
+  await app.listen(process.env.PORT);
```

Comment:
please include default port in case env.PORT is not provided.
```suggestion
  await app.listen(process.env.PORT | 3000);
```